base/setup/usetup/usetup.c

```diff
@@ -4144,6 +4146,8 @@ BootLoaderHarddiskMbrPage(PINPUT_RECORD Ir)
     NTSTATUS Status;
     WCHAR DestinationDevicePathBuffer[MAX_PATH];
 
+    MUIDisplayPage(BOOT_LOADER_HARDDISK_MBR_PAGE);
+
```

Comment:
Any chance for a single page with dynamic text depending on previously selected menu item?

Reply:
Sure thing.